initfiles/componentfiles/configxml/sparkThor.xsd

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+-->
+
+<!-- edited with XMLSPY v5 rel. 4 U (http://www.xmlspy.com) by Rajeev Moondhra (Seisint, Inc.) -->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified">
+    <xs:include schemaLocation="environment.xsd"/>
+    <xs:element name="SparkThorProcess">
+        <xs:annotation>
+            <xs:documentation>Describes a spark cluster</xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+            <!--DOC-Autobuild-code-->
+            <xs:annotation>
+                <xs:appinfo>
+                    <docid>Th.t1</docid>
+                </xs:appinfo>
+            </xs:annotation>  
+            <xs:sequence>
+                <xs:element name="Instance" maxOccurs="unbounded">
```

Comment:
It's hard to visualize, but if this is a container of instances, it needs to be removed (sparkthor will inherit its instances from the referenced thor cluster)